File: app/src/main/java/com/univreview/model/Setting.java
package com.univreview.model;

/**
 * Created by DavidHa on 2017. 2. 19..
 */
public class Setting extends com.univreview.model.model_kotlin.AbstractDataProvider {
    public long id;
    public String title;
    public String previewStr;

    public Setting(long id, String title) {
        this.id = id;
        this.title = title;
    }

    @Override
    public long getId() {
        return id;
    }

    @Override
    public void setId(long id) {
        this.id = id;
    }

    @Override
    public String getName() {
        return title;
    }

    @Override
    public void setName(String s) {
        this.title = s;
    }
}

File: app/src/main/java/com/univreview/model/Setting.kt
package com.univreview.model

import com.dhha22.bindadapter.Item

/**
 * Created by DavidHa on 2017. 8. 20..
 */
data class Setting(val id: Long,
                   val name: String,
                   var previewStr: String = "") : Item {
    constructor(id: Long, name: String) : this(id, name, "")
}